Makefile

```diff
@@ -31,7 +31,7 @@ build-all: assets build
 assets: go-bindata ui/bindata.go template/internal/deftmpl/bindata.go
 
 go-bindata:
-	-@$(GO) get -u github.com/jteeuwen/go-bindata/...
+	-@$(GO) get -u github.com/jteeuwen/go-bindata/go-bindata
```

Comment:
What was the issue with `...`?

Reply:
It just does not work anymore: https://github.com/golang/go/issues/29241